.github/workflows/release.yml

```diff
@@ -59,8 +59,17 @@ jobs:
           ./build/bin/nvim --version | head -n 3 >> $GITHUB_OUTPUT
           printf 'END\n' >> $GITHUB_OUTPUT
 
-  macOS:
-    runs-on: macos-14
+  macos:
+    strategy:
+      fail-fast: false
+      matrix:
+        runner: [ macos-12, macos-14 ]
+        include:
+          - runner: macos-12
+            arch: x86_64
+          - runner: macos-14
+            arch: arm64
```

Comment:
These are the default (only) arches for these runners, but it's good to be explicit here 👍